### PR TITLE
[FIX] account_invoice_constraint_chronology: msgmerge from .pot

### PR DESCRIPTION
**Pull request by petrus-v**
I'm not sure why .po files are unsynched on this module and not sure if we can handle this in an other way without PR ?

### CODE REVIEW

**Comment by petrus-v** [comment]
@hparfr @Kev-Roche do you have time to review this one for our customer ?!

**Comment by petrus-v** [comment]
@damdam-s could you gives some review here ?

File: account_invoice_constraint_chronology/i18n/account_invoice_constraint_chronology.pot [hunk 1]
```diff
@@ -19,6 +19,7 @@ msgid "Check Chronology"
 msgstr ""
 
 #. module: account_invoice_constraint_chronology
+#: code:addons/.src/account-financial-tools/account_invoice_constraint_chronology/model/account_move.py:0
```
Comment (hparfr) [inline account_invoice_constraint_chronology/i18n/account_invoice_constraint_chronology.pot]: is this path desired ?

Comment (petrus-v) [reply account_invoice_constraint_chronology/i18n/account_invoice_constraint_chronology.pot]: probably not :sweat_smile:

File: account_invoice_constraint_chronology/i18n/ar.po [hunk 2]
```diff
@@ -20,33 +20,62 @@ msgstr ""
...
 
 #. module: account_invoice_constraint_chronology
-#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_bank_statement_import_journal_creation__check_chronology
 #: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__check_chronology
 msgid "Check Chronology"
 msgstr ""
 
 #. module: account_invoice_constraint_chronology
-#: code:addons/account_invoice_constraint_chronology/model/account_invoice.py:66
+#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
```
Comment (hparfr) [inline account_invoice_constraint_chronology/i18n/ar.po:28]: do you know why the line number is wrong ?

File: account_invoice_constraint_chronology/i18n/ar.po [hunk 3]
```diff
@@ -20,33 +20,62 @@ msgstr ""
...
+msgstr "السجل اليومي"
+
+#. module: account_invoice_constraint_chronology
+#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal____last_update
+#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_move____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#~ msgid "Invoice"
+#~ msgstr "فاتورة"
```
Comment (petrus-v) [inline account_invoice_constraint_chronology/i18n/ar.po:81]: this is useless

File: account_invoice_constraint_chronology/i18n/ar.po (hunk 2 above)
Comment (petrus-v) [reply account_invoice_constraint_chronology/i18n/ar.po:28]: hum... good question probably related to my development environment...


Also I think the root matter here is that .pot file is still missing which is probably why it's not properly handle on weblate :thinking:

**Review by hparfr — APPROVED** [review]

File: account_invoice_constraint_chronology/i18n/ar.po (hunk 2 above)
Comment (hparfr) [reply account_invoice_constraint_chronology/i18n/ar.po:28]: ok

**Comment by petrus-v** [comment]
There are lot of modules in strange status in weblate on this repo https://translation.odoo-community.org/projects/account-financial-tools-14-0/ and in this  `account_invoice_constraint_chronology` module,  .pot terms are unsynched with weblate and  .po files:

 For instance in `account_invoice_constraint_chronology`  module:
*  in source code we have following term https://github.com/OCA/account-financial-tools/blob/14.0/account_invoice_constraint_chronology/model/account_move.py#L85-L91:
```python
            _(
                "Chronology conflict: An invoice with a higher number {highest_name}"
                " dated before {date_invoice} exists."
            ).format(
                highest_name=self._get_last_sequence(),
                date_invoice=format_date(self.env, self.invoice_date),
            )
```

* .pot file looks ok: https://github.com/OCA/account-financial-tools/blob/14.0/account_invoice_constraint_chronology/i18n/account_invoice_constraint_chronology.pot#L41:
```
msgid ""
"Chronology conflict: An invoice with a higher number {highest_name} dated "
"before {date_invoice} exists."
```
* in fr.po this msgid term can't be found.
* in weblate it looks like loaded terms are those from .po files


@OCA/accounting-maintainers (not sure if there is speciall team that maintains weblate? @oca-transbot ?) I'm not sure what should we do, when msgmerge should be run ? by developer or the CI ?

File: account_invoice_constraint_chronology/i18n/ar.po (hunk 2 above)
Comment (petrus-v) [reply account_invoice_constraint_chronology/i18n/ar.po:28]: infact .pot file looks good, the matter is probably about msgmerge not sure when it should be run between, I belives while merging PR before sending terms to weblate !?

**Pull request closed without merging**